### PR TITLE
chore: Update BOOTSTRAP_VERSION to 0.10

### PR DESCRIPTION
**Pull request by darcywong00**
This is to pull in the bootstrap changes from ~keymanapp/shared-sites#20.~ keymanapp/shared-sites#28

~The site doesn't load locally for me, but I haven't been able to troubleshoot why.~
Fixing up site_suffix gets the site to load now.

### CODE REVIEW

**Comment by mcdurdin** [comment]
> The site doesn't load locally for me, but I haven't been able to troubleshoot why.

Is this related to the bootstrap change? Don't want to merge this if so!

**Comment by darcywong00** [comment]
Ah my dev tier fails to load locally going from BOOTSTRAP_VERSION 0.4 (current master) to 0.6... Currently stumped

**Comment by mcdurdin** [comment]
> Ah my dev tier fails to load locally going from BOOTSTRAP_VERSION 0.4 (current master) to 0.6... Currently stumped

Let's rubber ducky this? Perhaps after issue triage?

**Comment by Meng-Heng** [comment]
Hello! Is this the cause of the styling errors with v0.4?
image

**Comment by darcywong00** [comment]
For me, v0.4 worked locally
v0.6+ had the styling issues

Though it worked fine for Marc. We haven't figured out why we're getting differences.

**Comment by mcdurdin** [comment]
> Hello! Is this the cause of the styling errors with v0.4?

This is happening because you haven't got s.keyman.com running locally -- you'll need to clone it and run it in docker as well.

File: inc/servervars.php [hunk 1]
```diff
@@ -32,9 +32,10 @@
     require_once __DIR__ . '/../cdn/deploy/cdn.php';
   }
 
-  $site_suffix = KeymanHosts::Instance()->Site_Suffix();
+  $site_suffix = KeymanHosts::Instance()->Tier() == KeymanHosts::TIER_DEVELOPMENT ?
+    ".localhost" : "";
```
Comment (darcywong00) [inline inc/servervars.php:36]: @Meng-Heng - I think this bit is why our local Docker setup wasn't working.

`KeymanHosts::Instance()->Site_Suffix()` was refactored out in the newer shared-sites version

Comment (Meng-Heng) [reply inc/servervars.php:36]: Thank you! I pulled the latest changes from s.keyman.com and deleted `KeymanHosts::Instance()->Site_Suffix()`, and now it works almost perfectly. 
image

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**